skymp5-functions-lib/index.ts

```diff
@@ -169,6 +171,13 @@ pointsByName.set('whiterun:safePlace', {
   rot: [0, 0, 176],
 });
 
+const startPoints = serverSettings["startPoints"];
+if(startPoints && typeof startPoints === 'object' && Array.isArray(startPoints)){
+  pointsByName.set('default', startPoints[0] as LocationalData);
```

Comment:
Please do NOT use `as` when you just want to silence the compiler. It results in having incorrect code compiled.
`as` is like `reinterpret_cast`. You just tell the compiler that everything ok but everything is not.

LocationalData and startPoints elements have different types. We need to convert carefully.

https://github.com/skyrim-multiplayer/skymp/blob/97864007140c21f12bca2df987da48adfdd384c7/skymp5-functions-lib/src/types/mp.d.ts#L90
https://github.com/skyrim-multiplayer/skymp/blob/c2f87b2547b6839308b8aa7dd8168d2bd4db7a3e/skymp5-server/ts/settings.ts#L14

Reply:
@Dinakrab **I see it will be too hard to fix so let's forget about startPoints and just use hall**